### DISCUSSION TITLE
Welcome to license-report Discussions!

### DISCUSSION THREAD

**Discussion by kessler** [ironSource/license-report#43] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tarunnagia** [comment]
Hi 
Is there a way to exclude some licenses from output?

**Reply by BePo65** [reply]
Sorry, there is no way to set some configuration parameters to exclude single licenses from output.

Which output format do you use? For output in the json format, you could use jq to remove the unwanted entries from the output (see [jq examples](https://gist.github.com/deepns/5b28ac586c31cf9d3cd8e4aedbbc425b) as a hint how to use jq).

**Reply by BePo65** [reply]
**update**
sorry for giving a wrong information. Working on a new feature, I stumbled over the corresponding lines of code.

So of course license-report can exclude some packages from the output. Just use the ['exclude' option](https://github.com/ironSource/license-report#exclude-packages).

**Comment by tarunnagia** [comment]
Hey, I think you were correct at first place! I was asking if we can exclude some licenses not packages.
Like I want to exclude all the packages from result json if their license is MIT but exclude works for excluding the packages.

**Reply by BePo65** [reply]
Oh, so it was not completely clear to me, what you mean by 'license', but this makes sense.  
In that case filtering with jq (or powershell, if you are on windows) is the best solution.
